api/millenium_patron.py

```diff
@@ -63,6 +64,18 @@ class MilleniumPatronAPI(BasicAuthenticationProvider, XMLParser):
     AUTHENTICATION_MODES = [
         PIN_AUTHENTICATION_MODE, FAMILY_NAME_AUTHENTICATION_MODE
     ]
+
+    SETTINGS = [
+        { "key": VERIFY_CERTIFICATE, "label": _("Verify Certificate"), "optional": True },
```

Comment:
I left it the same internally but changed the labels so it will hopefully be more clear in the admin interface.